lang/common-lisp/avro.asd

```diff
@@ -0,0 +1,248 @@
+;;; Copyright 2024 Google LLC
```

Comment:
I am not sure but I think this line needs to be removed if this PR will be accepted

Reply:
I updated the license headers to match the other implementations